src/GeniusYield/Types/Providers.hs

```diff
@@ -317,27 +324,28 @@ makeSlotActions :: NominalDiffTime
                 -- ^ Getting current slot directly from the provider
                 -> IO GYSlotActions
 makeSlotActions t getSlotOfCurrentBlock = do
-    slotRefetchTime <- addUTCTime t <$> getCurrentTime
+    getTime         <- mkAutoUpdate defaultUpdateSettings {updateAction = getCurrentTime}
```

Comment:
Maybe we should note something down (like a TODO or an issue) to investigate whether the auto update thing is better than just using `getCurrentTime`?

Reply:
Yes, this was on radar! I have created issue for it [here](https://github.com/geniusyield/atlas/issues/339).